lib/rules/display-name.js

```diff
@@ -116,7 +117,7 @@ module.exports = {
       );
 
       const namedFunctionExpression = (
-        (node.type === 'FunctionExpression' || node.type === 'ArrowFunctionExpression') &&
+        astUtil.isFunctionLikeExpression(node) &&
```

Comment:
I think this kind of change could be peeled off into a separate pr, and landed first?

Reply:
Ah, you’re right 👍